packages/blade/src/components/Icons/_Svg/Svg/Svg.web.tsx

```diff
@@ -1,19 +1,30 @@
 import type { ReactElement } from 'react';
 import type { SvgProps } from './types';
-import { metaAttribute, makeAccessible, MetaConstants } from '~utils';
+import { metaAttribute, makeAccessible, MetaConstants, getStyledProps } from '~utils';
+import BaseBox from '~components/Box/BaseBox';
 
-const Svg = ({ children, height, viewBox, width, fill }: SvgProps): ReactElement => {
+const Svg = ({
+  children,
+  height,
+  viewBox,
+  width,
+  fill,
+  ...styledProps
+}: SvgProps): ReactElement => {
   return (
-    <svg
-      {...makeAccessible({ hidden: true })}
-      {...metaAttribute(MetaConstants.Component, MetaConstants.Icon)}
-      height={height}
-      viewBox={viewBox}
-      width={width}
-      fill={fill}
-    >
-      {children}
-    </svg>
+    // line-height adds some extra spacing in div around svg so removing it
+    <BaseBox display="inline-block" lineHeight="spacing.0" {...getStyledProps(styledProps)}>
```

Comment:
Is there some default lineHeight already that's causing this issue? Won't it be 0 by default anyway?